playground/infrastructure/grpc_client.py

```diff
@@ -34,7 +34,7 @@ class GRPCClient:
 
     def __init__(self, wait_for_ready=True):
         use_webgrpc = os.getenv("BEAM_USE_WEBGRPC", False)
-        timeout = os.getenv("GRPC_TIMEOUT", 10)
+        timeout = int(os.getenv("GRPC_TIMEOUT", 10))
```

Comment:
I think we should not hard code const keys in the code but hoist them using for example
```go
const (
    grpcTimeoutKey = "GRPC_TIMEOUT"
)
```

Reply:
Files a task to do this: https://github.com/apache/beam/issues/25202. This change seems to be outside of the scope of this PR.